operator/controllers/seldondeployment_controller.go

```diff
@@ -170,6 +170,46 @@ func createIstioResources(mlDep *machinelearningv1.SeldonDeployment,
 			return nil, nil, err
 		}
 	}
+
+	getDefaultHttpRoute := func() istio_networking.HTTPRoute {
+		route := istio_networking.HTTPRoute{
+			Match: []*istio_networking.HTTPMatchRequest{
+				{
+					Uri: &istio_networking.StringMatch{MatchType: &istio_networking.StringMatch_Prefix{Prefix: "/seldon/" + namespace + "/" + mlDep.Name + "/"}},
+				},
+			},
+			Rewrite: &istio_networking.HTTPRewrite{Uri: "/"},
+		}
+
+		if istioRetries > 0 {
+			route.Retries = &istio_networking.HTTPRetry{Attempts: int32(istioRetries), PerTryTimeout: &types2.Duration{Seconds: int64(istioRetriesTimeout)}, RetryOn: "gateway-error,connect-failure,refused-stream"}
+		}
+
+		return route
+	}
+
+	getDefaultGrpcRoute := func() istio_networking.HTTPRoute {
```

Comment:
```suggestion
	getDefaultGRPCRoute := func() istio_networking.HTTPRoute {
```

Reply:
useing Grpc,  becasue whole project is useing Grpc in other place, so I think we should keep it.